test/e2e/framework/ssh/ssh.go

```diff
@@ -24,11 +24,29 @@ import (
 	"path/filepath"
 	"time"
 
+	"github.com/onsi/gomega"
 	"golang.org/x/crypto/ssh"
 	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/fields"
 	"k8s.io/apimachinery/pkg/util/wait"
 	clientset "k8s.io/client-go/kubernetes"
 	sshutil "k8s.io/kubernetes/pkg/ssh"
+	e2elog "k8s.io/kubernetes/test/e2e/framework/log"
```

Comment:
I am not a fan of aliasing everything. There should be good and exceptional reasons to override the package name.